packages/typography/src/typography.scss

```diff
@@ -0,0 +1,91 @@
+@use "../external/utilities" as *;
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap");
+
+@include sui-class($rtl: false, $theme: null) {
+	* {
+		font-family: "Roboto", sans-serif;
+		font-variant-ligatures: none;
+		text-rendering: optimizeLegibility;
+		-webkit-font-smoothing: antialiased;
+		-moz-osx-font-smoothing: grayscale;
+		text-shadow: rgba(0, 0, 0, 0.01) 0 0 1px;
+	}
+
+	p {
```

Comment:
@iamleigh We have two variations for body size. I think we missed a small one here.

Reply:
Nice catch! I will add it right away.